### PR TITLE
Added new code for Jpyter notebook, basic usage, command line and docker.

### PR DESCRIPTION
**Pull request by hackit-coder**
I this commits, I have added the basic structure for importing and using Taipy in Command line, Jpyter notebook, Docker and basic usage along with typos fix.

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -25,6 +25,34 @@ pipenv install taipy
...
+
+# Create a scenario
+scenario = Scenario(
+    config=config,
+    id="my_scenario"
+)
+
+# Run the scenario
+scenario.run()
+```
```
Comment (trgiangdo) [inline INSTALLATION.md]: This is not how to use Taipy. This example won't work

File: INSTALLATION.md [hunk 2]
```diff
@@ -47,8 +75,33 @@ git clone https://github.com/Avaiga/taipy.git
...
+import taipy as tp
+
+# Create a configuration
+config = tp.Config(...)
+
+# Create a scenario
+scenario = tp.Scenario(config)
+
+# Run the scenario
+scenario.run()
```
Comment (trgiangdo) [inline INSTALLATION.md]: This example won't work

File: INSTALLATION.md [hunk 3]
```diff
@@ -47,8 +75,33 @@ git clone https://github.com/Avaiga/taipy.git
...
+```
+## Running Taipy from Command Line
+
+```
+bash
+taipy run --scenario-id my_scenario
```
Comment (trgiangdo) [inline INSTALLATION.md]: This command won't work

File: README.md [hunk 4]
```diff
@@ -16,7 +16,7 @@ Build Python Data & AI web applications
 
 <div align="center">
 From simple pilots to production-ready web applications in no time. <br />
-No more compromise on performance, customization, and scalability.
+No more compromise on performance, customization and scalability.
```
Comment (trgiangdo) [inline README.md:19]: The comma here is normal, there's no need to remove it

**Review by trgiangdo — CHANGES_REQUESTED** [review]
Hello @hackit-coder, thank you for interested in contributing to Taipy.

The changes doesn't seem all fit to the project. Please take a look at my reviews and let me know what you think

**Comment by hackit-coder** [comment]
As you said, the code is wrong, I deleted the commits and will work on them later, but aI think the Readme.md file really needs some decoration so plz look into that commit.

File: README.md [hunk 5]
```diff
@@ -33,19 +33,19 @@ No more compromise on performance, customization, and scalability.
     <br />
     <a href="https://discord.com/invite/SJyz2VJGxV"><strong>  🫱🏼‍🫲🏼 Discord support </strong></a>
     <br />
-    <a href="https://docs.taipy.io/en/latest/gallery/"><strong> 👀 Demos & Examples </strong></a>
+    <a href="https://docs.taipy.io/en/latest/gallery/"><strong> 👀Want some Demos & Examples </strong></a>
```
Comment (trgiangdo) [inline README.md:36]: This is just a link to the Demos page. I would keep it short as it is

File: README.md [hunk 6]
```diff
@@ -33,19 +33,19 @@ No more compromise on performance, customization, and scalability.
...
 
 Taipy is designed for data scientists and machine learning engineers to build data & AI web applications.
 &nbsp;
 
-⭐️ Enables building production-ready web applications. <br />
-⭐️ No need to learn new languages. Only Python is needed.<br />
-⭐️ Concentrate on Data and AI algorithms without development and deployment complexities.<br />
+⭐️->Enables building production-ready web applications. <br />
+⭐️-> No need to learn new languages. Only Python is needed.<br />
+⭐️-> Concentrate on Data and AI algorithms without development and deployment complexities.<br />
```
Comment (trgiangdo) [inline README.md]: The ⭐️ is already acting like a bullet. I don't understand the purpose of the "->". Can you explain?

File: README.md [hunk 7]
```diff
@@ -116,7 +116,7 @@ You can use the Taipy Studio extension in Visual Studio Code to configure your s
 Your configuration is automatically saved as a TOML file.<br />
 Check out Taipy Studio [Documentation](https://docs.taipy.io/en/latest/manuals/studio/)
 
-For more advanced use cases or if you prefer coding your configurations instead of using Taipy Studio,<br />
+For more advanced use cases or if you want to make your own configurations instead of using Taipy Studio,<br />
```
Comment (trgiangdo) [inline README.md:119]: There is a difference between "make configuration" and "coding configuration". With Taipy Studio, you can do configuration without coding.

**Review by trgiangdo — COMMENTED** [review]
You completely delete the INSTALATION.md file though.

Anyway, I do have some comments and questions on the README changes.

**Review by jrobinAV — COMMENTED** [review]
I would keep nothing. I let @RymMichaut decide.

**Comment by RymMichaut** [comment]
> I would keep nothing. I let @RymMichaut decide.

Same as @jrobinAV, I would keep nothing and leave it as it is. 
@trgiangdo has said everything 🙏🏻

**Pull request closed without merging**